lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -145,6 +145,7 @@ def create_setup_intent(payment_method, options = {})
             post[:on_behalf_of] = options[:on_behalf_of] if options[:on_behalf_of]
             post[:usage] = options[:usage] if %w(on_session off_session).include?(options[:usage])
             post[:description] = options[:description] if options[:description]
+            post[:expand] = ['latest_attempt']
```

Comment:
What is this parameter?

Reply:
it expands certain parameters in Stripe we use it a lot but I just want to make sure it won't change any response fields on our side

Reply:
> it expands certain parameters in Stripe to get more information. We use it a lot but I just want to make sure it won't change any response fields on our side other than what we are trying to accomplish here

Reply:
Where else do we use it @almalee24? Would we want to expand this for all requests, not just setup intents?

Reply:
In stripe's response they generally send something like this. (Along with all the other parameters) `{"latest_attempt": "setatt_1Omn1KAWOtgoysoge1eoODtu"}` . The `expand` param pulls all the data from the latest_attempt object into the response so that you don't have to make a separate call to stripe to retrieve it.

Reply:
I looks like at least for setup_intent `latest_attempt` only returns a string but with this expand it not returns a object. The object is what we expect on our side because we use that to grab NTID and other stuff

Reply:
I think this is a safe and good addition to setup intent. I'm surprised that it currently just returns a string on setup intent but I wonder if it's only sometimes or all the time

Reply:
Also `latest_attempt` is just a Setup Intent parameter so it should only be added here. Since we are do this change it would be nice to cleaning up Core a bit for the StripePI NTID since after this change we will be returning it on all SetupIntent transactions now

Reply:
`latest_attempt` is only a field for the `setup_intent` object so we should be good with having it only there.